test/methods.jl

```diff
@@ -154,12 +129,12 @@ import FiniteDifferences: estimate_magitude, estimate_roundoff_error
         end
     end
 
-    @testset "Derivative of cosc at 0 (#124)" begin
-        @test central_fdm(5, 1)(cosc, 0) ≈ -(pi ^ 2) / 3 atol=1e-9
-        @test central_fdm(10, 1, adapt=3)(cosc, 0) ≈ -(pi ^ 2) / 3 atol=5e-14
+    @testset "Derivative of cosc at 0 (issue #124)" begin
```

Comment:
Ah, I see that this is tested here. Is there a reason not to included it in your standard test suite?